scripts/needs_testrun.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
```

Comment:
should we add some doctests?

would be interesting to add one for `get_base_branch` since if we can use a static `PR_NUMBER`, then as long as the HTML/CSS doesn't change, the expected result should be consistent. could be a good litmus test to catch if something changes/stops working.

same idea for `get_changd_files`?

this isn't a hard requirement, just a suggestion, feel free to resolve if you think it is not needed.

Reply:
good idea!